File: APIdoc/js/tutorials/listFoldersPreviewFile.js
var email = ""
var password = ""
var apiKey = ""

var sessionKey = auth(email, password, apiKey)
var rules = listAccessRules(sessionKey)
var folders = rules[2]
for(var i = 0; i < folders.length; i++){
	if(folders[i][2] == "FILE"){
		var preview = filePreview(sessionKey, folders[i][16], folders[i][0], folders[i][1])
		console.log(preview)
	}
}

File: APIdoc/ruby/tutorials/listFoldersPreviewFile.rb
require_relative '../APIDoc.rb'

email, password, apiKey = ARGV[0], ARGV[1], ARGV[2]

apiDoc = APIDoc.new(email, password, apiKey)

sessionKey = apiDoc.sessionKey()

accessRules = apiDoc.listAccessRules(sessionKey)

folders = apiDoc.listFolders(sessionKey, accessRules[2][1])

for f in folders
	if (f[2] == "FILE")
		preview = apiDoc.filePreview(sessionKey, f[16], f[0], f[1])
		print(preview)
	end		
end